pptx_blueprint/__init__.py

```diff
@@ -85,8 +90,42 @@ def _find_shapes_in_slide(slide):
     def save(self, filename: _Pathlike) -> None:
         """Saves the updated pptx to the specified filepath.
 
-        Args:
+         Args:
             filename (path-like): file name or path
         """
         # TODO: make sure that the user does not override the self._template_path
         pass
+
+    def save_pdf(self, file_path: _Pathlike) -> None:
+        """Exports the updated pptx to the specified filepath as pdf file.
+
+        Args:
+            filename (path-like) file name or path
+        """
+
+        try:
+            subprocess.Popen(['libreoffice', '--version'],
+                             stdout=subprocess.DEVNULL)  # check if libreoffice is installed
+
+            path = Path(file_path)
+            outdir = path.parent
+            file_name = path.name
+
+            # create temporary directory for pptx
+            os.path.exists('tmp') or os.mkdir('tmp')
```

Comment:
Better to use `tempfile.TemporaryDirectory()` as a context (https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory).
~~~
with tempfile.TemporaryDirectory() as tmpdirname:
    ...
~~~

This encapsulates all the temp file logic. (For example  the current proposal tries to delete `tmp` even if it was already there before the call to `save_pdf()`).

Reply:
thanks for suggestion.